lib/blockscore/errors/blockscore_error.rb

```diff
@@ -0,0 +1,32 @@
+module BlockScore
+  class BlockScoreError < StandardError
```

Comment:
this should probably just be `BlockScore::Error` since we already have the namespace